entity-framework/core/miscellaneous/spatial.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Spatial Data - EF Core
+author: bricelam
+ms.author: bricelam
+ms.date: 11/01/2018
+uid: core/miscellaneous/spatial
```

Comment:
Any reasons you think this should be under miscellaneous instead of modeling?

Reply:
It's a bit of everything--reverse engineering, modelling, querying, using NTS...

But I'm happy with it wherever.

Reply:
Yes, you can do all those things with spatial, but isn’t that true for most things under modeling? I have the impression that the ability to map a type or concept in the model is the defining characteristic here.

Reply:
Ok. Thank you for letting me know.